dms/models/dms_access_groups.py

```diff
@@ -121,24 +157,14 @@ def _compute_inclusive_permissions(self):
         for one in self:
             one.update(
                 {
-                    "perm_inclusive_%s"
-                    % perm: (
+                    "perm_inclusive_%s" % perm: (
                         one["perm_%s" % perm]
                         or one.parent_group_id["perm_inclusive_%s" % perm]
                     )
                     for perm in ("create", "unlink", "write")
                 }
             )
 
-    @api.model
```

Comment:
Please reduce the noise in the migration commit. Do it if you want on a REF one, but again, I advise you to not do it on code that is not yours, as everyone has a different style, and we should focus on the things that brings value to the community.